server/connectors/spinnaker_connector/client.py

```
@@ -243,6 +245,19 @@ def close(self) -> None:
     def _request(self, method: str, path: str, **kwargs: Any) -> Any:
         """Send an authenticated request to Spinnaker Gate API."""
         kwargs.setdefault("timeout", SPINNAKER_TIMEOUT)
+        if not path.startswith("/"):
+            raise SpinnakerAPIError("Invalid API path: must start with /")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)